components/common/splashPageSearchBar.tsx

```diff
@@ -133,8 +133,8 @@ export const SplashPageSearchBar = (props: SearchProps) => {
                 {...params.inputProps}
                 type="search"
                 id="mainSearch"
-                className="rounded-md border-primary-dark border-2 w-full h-12 pl-12 bg-white text-primary-dark placeholder-primary-dark font-bold"
-                placeholder="Search section number, professor name, course number...."
+                className="rounded-md border-primary-dark border-2 w-full h-12 pl-12 bg-white text-primary-dark placeholder-primary-light font-bold"
+                placeholder="CS1337..."
```

Comment:
This probably does not need to be changed to match the figma exactly, previous placeholder was fine too